### DISCUSSION TITLE
객체를 접근하는 권한

### DISCUSSION THREAD

**Discussion by stopmin** [BDD-CLUB/challenge-every-week-posting#477] [1. 개발 관련 글 모음]
안녕하세욤! 요즘 글쓰는게 힘드네요 ㅠㅠ
카테캠 생각보다 강의가 많기도 하고 알고리즘 벽부수기 문제 김태윤씨 따라 풀다가 메모리 이슈로 아직 해결을 못했습니닷 하하 그래서 석이 조금 나간 상태네요 ㅎㅎ 반드시 풀고 글 올리겠습니당!!!!

이번주차는 카테캠 강의내용 정리한 것 업로드 합니당 ..!!

[블로그 바로가기](https://stopmin.tistory.com/entry/%EA%B0%9D%EC%B2%B4%EB%A5%BC-%EC%A0%91%EA%B7%BC%ED%95%98%EB%8A%94-%EA%B6%8C%ED%95%9C-%EC%9D%B4%ED%95%B4%ED%95%98%EA%B8%B0)

**Comment by 02ggang9** [comment]
잘 봤습니다~ 카테캠 ㅎㅇㅌ 하세용
